src/components/implementation/no_interface/scheddev/sl.h

```diff
@@ -174,10 +192,23 @@ sl_cs_exit(void)
 	if (!ps_cas(&sl__globals()->lock.u.v, cached.v, 0))    goto retry;
 }
 
+static inline microsec_t
+sl_cyc2usec(cycles_t cyc)
+{ return cyc / sl__globals()->cyc_per_usec; }
+
+static inline microsec_t
+sl_usec2cyc(microsec_t usec)
+{ return usec * sl__globals()->cyc_per_usec; }
+
 static inline cycles_t
 sl_now(void)
 { return ps_tsc(); }
 
+static inline microsec_t
+sl_now_usec(void)
+{ return sl_cyc2usec(sl_now()); }
+
```

Comment:
I guess I made a mistake when replying to @phanikishoreg earlier, my apologies. This is just a small helper method I added, and is the only code change unrelated to locks in this commit. (Although it is/was used in the library I think.